test/README.md

```diff
@@ -0,0 +1,28 @@
+
+# libecalc test directory
+
+This directory contains a convenience lightweight Dockerfile to run tests for libecalc. The original reason
+behind this need is to run snapshot tests in an x86_64 environment, since developer may run on ARM64 architecture.
+Some of the snapshot tests are architecture dependent, and running them on an ARM64 machine will result in
+different values than for x86_64. This is due to Delanay triangulation algorithm used in some of the
+compressor calculations.
+
+## Running tests
+
+Since running tests in a container is only required for some of the tests, the default setup is already
+configured in *docker-compose.yml* file. To run tests, simply run:
+
+```bash
+docker compose up
```

Comment:
```suggestion
docker compose run test
```

instead, to make it clearer that we just run the one service? Also, I support your suggestion to add a .sh file/script instead, which is probably more intuitive.